jina/peapods/runtimes/container/__init__.py

```diff
@@ -163,6 +164,36 @@ def _docker_run(self, replay: bool = False):
                     'mode': 'rw',
                 }
 
+        device_requests = []
+        if self.args.gpus:
```

Comment:
maybe u can put this in an static method where u can do better unittesting of this logic